operator/gitops/argocd/pipeline-service/openshift-pipelines/pipelines-scc.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: pipelines-scc
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities:
+  - SETFCAP
+defaultAddCapabilities: null
+fsGroup:
+  type: MustRunAs
+groups:
+  - system:cluster-admins
+priority: 10
```

Comment:
another addition from `restricted` SCC

Reply:
this affects the sort order for applying SCCS

this tells openshift to consider this SCC before the default SCCs